shared/api/resource.go

```diff
@@ -551,6 +554,19 @@ type ResourcesNetworkCardSRIOV struct {
 	VFs []ResourcesNetworkCard `json:"vfs" yaml:"vfs"`
 }
 
+// ResourceNetworkCardVDPA represents the VDPA configuration of the network card
+//
+// swagger:model
+//
+// API extension: resources_v2.
```

Comment:
Is this API extension right? I think it should be its own separate API extension.

Reply:
Should be `ovn_nic_acceleration_vdpa` I think.